src/App.tsx

```diff
@@ -1,14 +1,21 @@
-import React from "react";
+import React, { useState } from "react";
 import { RecipesProvider } from "./modules/recipes/RecipesProvider";
 import "./App.css";
 import { AppRouter } from "./AppRouter";
 import { UserProvider } from "./modules/user/UserContext";
 import Container from "@mui/material/Container";
-import { NavBar } from "./shared-component/Navbar";
+import { NavBar } from "./shared-component/navbar/index";
```

Comment:
When you import from an index.js file in a folder, you don't need to specify the index part:
```suggestion
import { NavBar } from "./shared-component/navbar";
```